tournaments/index.js

```diff
@@ -144,6 +159,41 @@ class Tournament {
 		return true;
 	}
 
+	getBanlist() {
+		let bans = [];
+		let unbans = [];
+		let enforces = [];
+		let removes = [];
+		for (let i = 0; i < this.banlist.length; i++) {
+			let ban = this.banlist[i];
+			let unban;
+			if (ban.charAt(0) === '!') {
+				unban = true;
+				ban = ban.substr(1);
+			}
+			if (ban.startsWith('Rule:')) {
+				ban = ban.substr(5);
+				if (unban) {
+					removes.push(ban);
+				} else {
+					enforces.push(ban);
+				}
+			} else {
+				if (unban) {
+					unbans.push(ban);
+				} else {
+					bans.push(ban);
+				}
+			}
+		}
+		let html = [];
+		if (bans.length) html.push("<b>Bans</b> - " + Chat.escapeHTML(bans.join(", ")));
```

Comment:
> I think this is way too much mental effort to spend on something that has zero effect on accessibility, text parsability, or anything like that. So that's why I think we should just use `<b>` and `<i>` for everything.

https://github.com/Zarel/Pokemon-Showdown/pull/3247#issuecomment-280903156